### PR TITLE
[MEJB-130] Accept ejbVersion 4.x

### PR DESCRIPTION
**Pull request by pzygielo**
To make clear that you license your contribution under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) you have to acknowledge this by using the following check-box.

 - [X] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

### CODE REVIEW

**Comment by pzygielo** [comment]
May I ask for review, please?

**Comment by andreastedile** [comment]
I would be interested for EJB version 4 too.

File: src/main/java/org/apache/maven/plugins/ejb/EjbMojo.java [hunk 1]
```diff
@@ -465,10 +465,10 @@ private File generateEjbClient()
     private void checkEJBVersionCompliance( File deploymentDescriptor )
         throws MojoExecutionException
     {
-        if ( !ejbVersion.matches( "\\A[2-3]\\.[0-9]\\z" ) )
+        if ( !ejbVersion.matches( "\\A[2-4]\\.[0-9]\\z" ) )
```
Comment (elharo) [inline src/main/java/org/apache/maven/plugins/ejb/EjbMojo.java:468]: a unit test would be easier to manage than an integration test

Comment (pzygielo) [reply src/main/java/org/apache/maven/plugins/ejb/EjbMojo.java:468]: > a unit test would be easier to manage than an integration test

IT replaced with UTs.

**Review by elharo — APPROVED** [review]

**Comment by elharo** [comment]
Running through Jenkins: https://ci-builds.apache.org/job/Maven/job/maven-box/job/maven-ejb-plugin/job/mejb-130/

**Comment by jira-importer** [comment]
Resolve #117

**Pull request merged**